### PR TITLE
fix ArgumentOutOfRangeException in TableRenderer

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Kryptos-FR** [comment]
Hi @TobBrandt-Work could add a bit more details in the description.

If possible, include a [minimum reproducible case](https://stackoverflow.com/help/minimal-reproducible-example).

Thanks a lot.

**Comment by ghost** [comment]
You need to use the Pipe Tables extension with these options:

```
pipeline.UsePipeTables(new PipeTableOptions { RequireHeaderSeparator = false })
```
then pass the following markdown as input to the parser:

```
|||:X
```

This creates a table with an empty list of column definitions, which in turn causes the exception in `TableRenderer`.

Maybe this is a bug in the parser?

**Comment by Kryptos-FR** [comment]
Thanks for the details. I'll have a look at it in more depth on the week-end.

**Comment by Kryptos-FR** [comment]
Looking at markdig code itself, the same fix was done some time ago, exactly to support that option (https://github.com/lunet-io/markdig/commit/2147e434f79cf562dd72893202092d7f478ea649).

So good spot and thanks for the fix.

Edit: I have made a small suggestion. Could you apply it? Alternatively you can allow changes to be pushed to your pull-requests.

File: src/Markdig.Wpf/Renderers/Wpf/Extensions/TableRenderer.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ protected override void Write([NotNull] WpfRenderer renderer, [NotNull] Table ta
                     renderer.Write(cell);
                     renderer.Pop();
 
-                    if (table.ColumnDefinitions != null)
+                    if (table.ColumnDefinitions != null && table.ColumnDefinitions.Count > 0)
```
Comment (Kryptos-FR) [inline src/Markdig.Wpf/Renderers/Wpf/Extensions/TableRenderer.cs]:
```suggestion
                    if (table.ColumnDefinitions.Count > 0)
```
`table.ColumnDefinitions` is guaranteed to be non-null at this point.

**Comment by ghost** [comment]
Done

**Pull request merged**